src/devices/machine/ram.cpp

```diff
@@ -2,19 +2,20 @@
 // copyright-holders: Dirk Best
 /*************************************************************************
 
-    RAM device
+    ram.cpp
```

Comment:
I prefer the type of device here instead of the filename, which can change (we have a lot of incidents in MAME where the filename specified in the header isn't the real filename anymore). I also don't see much value in duplicating the filename.